xml/System.IO/Path.xml

```diff
@@ -2310,7 +2310,7 @@ The method does not verify that the path exists or test to see if the current pr
 
 4.  The Windows directory.
 
-In .NET 7 and later, when the process calling the method is running as `SYSTEM`, the method skips the preceding sequence and returns `C:\Windows\SystemTemp`. This location is inaccessible to processes not running as `SYSTEM`.
+On Windows 10 and Server Build 20348 and later, when the process calling the method is running as `SYSTEM`, the method skips the preceding sequence and returns `C:\Windows\SystemTemp`. This location is inaccessible to processes not running as `SYSTEM`.
```

Comment:
Also mentioning .NET 7 and newer, as well as .NET 6 starting with 6.0.32 would be helpful in my opinion, as otherwise it is unclear that in older OS Versions those runtimes also behave that way.

Reply:
@gewarren what is the recommended approach for situations where we have change the behavior of the product starting in specific version? Should we list all affected versions?

Reply:
Does .NET Framework provide an opt-out switch for compatibility?

Reply:
```suggestion
In .NET 7 and later versions (or .NET 6 version 6.0.32 and later), OR if your operating system is Windows 10 or Windows Server build 20348 or later and you're using any .NET version, when the process calling the method is running as `SYSTEM`, the method skips the preceding sequence and returns `C:\Windows\SystemTemp`. This location is inaccessible to processes not running as `SYSTEM`.
```

Does this convey the message? I think it's helpful to be as explicit as possible.

Reply:
Yes, this is very precise, thank you!

Reply:
Maybe rewording it to 

> and you're using any .NET **Framework** version

would be more accurate, as using 6.0.31 on newer (or any) Windows builds does not contain the security fix.

Reply:
> Does .NET Framework provide an opt-out switch for compatibility?

On 4.7/4.8 it evals an environment variable called "COMPlus_Disable_GetTempPath2" to revert to the old behavior if set to "true".